test/html.test.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const assert = require('assert');
+const jsdom = require('jsdom').jsdom;
+const html = require('../lib/html');
+
+describe('html', function() {
+  before(function() {
+    global.document = jsdom();
+  });
+
+  after(function() {
+    delete global.document;
+  });
+
+  describe('raw', function() {
+    it('should export a method', function() {
+      assert.equal(typeof html.raw, 'function');
+    });
+
+    it('should create a new Markup instance', function() {
+      const inst = html.raw();
+      assert.equal(inst.constructor.name, 'Markup');
+    });
+
+    describe('Markup instance', function() {
+      it('should store an internal value', function() {
+        const inst = html.raw('foo');
+        assert.equal(inst.value, 'foo');
+      });
+
+      it('should implement a toString method and output value', function() {
+        const inst = html.raw('foo');
+        assert.equal(inst.toString(), 'foo');
+      });
+    });
+  });
+
+  describe('isRaw', function() {
+    it('should export a method', function() {
+      assert.equal(typeof html.isRaw, 'function');
+    });
+
+    it('should identify raw Markup instances', function() {
+      const inst = html.raw('foo');
+      assert.equal(html.isRaw(inst), true);
```

Comment:
When doing truthy/falsey checks, I prefer `assert.ok` _unless_ my intent is strict equal. In which case I'd use `assert.strictEqual`. I do this instead of any `assert.equal(..., true|false)` because JS's boolean `==` will often do unusual things. Intead of `assert.equal(x, false)`, I'll do `assert.ok(!x)` because there's no `assert.notOk`.

Reply:
good point. I'll change most of these to `strictEqual` since I want to be as strict as possible. I absolutely want a `boolean` here